### PR TITLE
PM Deploy CSV updates

### PR DESCRIPTION
**Pull request by snwhite**
Updates to CP01CNPM, CPO03ISPM, and CP04OSPM from Metadata Review.

### CODE REVIEW

File: deployment/CP03ISPM_Deploy.csv [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+AR-31,,AR-34,,CP03ISPM-WFP01-02-DOFSTK000,2,1,2018-10-23T02:55:00,2019-04-17T11:39:00,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40.36258,-70.8787,,68,92,
+AR-31,,AR-34,,CP03ISPM-WFP01-03-CTDPFK000,2,1,2018-10-23T02:55:00,2019-04-17T11:39:00,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.36258,-70.8787,,68,92,
+AR-31,,AR-34,,CP03ISPM-WFP01-04-FLORTK000,2,1,2018-10-23T02:55:00,2019-04-17T11:39:00,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.36258,-70.8787,,68,92,
+AR-31,,AR-34,,CP03ISPM-WFP01-05-PARADK000,2,1,2018-10-23T02:55:00,2019-04-17T11:39:00,CGMCP-03ISPM-00002,CGVEH-CWFPCP-30001,CGINS-PARADK-20541,40.36258,-70.8787,,68,92,
 AR-39,,,,CP03ISPM-RII01-02-ADCPTG010,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,,CGINS-ADCPTG-18660,40.364817,-70.888668,,69,92,
 AR-39,,,,CP03ISPM-SBS01-00-STCENG000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,,CGCON-BSTC01-00018,40.364817,-70.888668,,0,92,
 AR-39,,,,CP03ISPM-SBS01-01-MOPAK0000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,,CGINS-MOPAK0-58481,40.364817,-70.888668,,0,92,
```
Comment (rtravis-whoi) [inline deployment/CP03ISPM_Deploy.csv]: looks good

File: deployment/CP04OSPM_Deploy.csv [hunk 2]
```diff
@@ -1,80 +1,80 @@
...
+AT-31,,AR-04,,CP04OSPM-WFP01-01-VEL3DK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-VEL3DK-00022,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-02-DOFSTK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-DOFSTK-02397,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-03-CTDPFK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-CTDPFK-00106,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-04-FLORTK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-FLORTK-00969,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-05-PARADK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-PARADK-20423,39.93565,-70.87998,,412,458,
+AR-04,,AR-08,,CP04OSPM-SBS01-00-STCENG000,5,1,2016-05-19T22:35:00,2016-09-28T13:25:00,CGMCP-04OSPM-00005,,CGCON-BSTC01-00004,39.94325,-70.88667,,0,438,
+AR-04,,AR-08,,CP04OSPM-SBS11-02-MOPAK0000,5,1,2016-05-19T22:35:00,2016-09-28T13:25:00,CGMCP-04OSPM-00005,,CGINS-MOPAK0-11357,39.94325,-70.88667,,0,438,
```
Comment (rtravis-whoi) [inline deployment/CP04OSPM_Deploy.csv]: The config shows MOPAK s/n as 6223-4220-58484.  Other OSPM deployments do have 11357, but not sure if correct for deployment 5.  Metadata review sheet doesn't actually note what the sn should have been changed to, so I'm not sure if config was not updated or if this should match config sn

File: deployment/CP04OSPM_Deploy.csv [hunk 3]
```diff
@@ -84,13 +84,13 @@ AR-31,,AR-34,,CP04OSPM-WFP01-04-FLORTK000,10,1,2018-11-06T14:36:00,2019-04-22T16
...
+AR-34,,AR-39,,CP04OSPM-WFP01-02-DOFSTK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.9435,-70.8862,,414,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-03-CTDPFK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.9435,-70.8862,,414,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-04-FLORTK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.9435,-70.8862,,414,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-05-PARADK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.9435,-70.8862,,414,435,
 AR-39,,,,CP04OSPM-RII01-02-ADCPSL010,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,,CGINS-ADCPSL-21393,39.935558,-70.880328,,406,451,
 AR-39,,,,CP04OSPM-SBS01-00-STCENG000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,,CGCON-BSTC01-00012,39.935558,-70.880328,,0,451,
 AR-39,,,,CP04OSPM-SBS11-02-MOPAK0000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,,CGINS-MOPAK0-11347,39.935558,-70.880328,,0,451,
```
Comment (rtravis-whoi) [inline deployment/CP04OSPM_Deploy.csv]: One discrepancy to address for OSPM deployment 5 regarding the MOPAK UID that the csv was changed to.  See specific comment attached to that line in PR.

File: deployment/CP04OSPM_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/CP04OSPM_Deploy.csv]: Based on working with John Lund, we believe the MOPAK for OPSM-5 is 6223-4220-11357.  The Config Sheet will need to be corrected.

Comment (rtravis-whoi) [reply deployment/CP04OSPM_Deploy.csv]: Roger that - config has been updated to reflect MOPAK 6223-4220-11357.

File: deployment/CP04OSPM_Deploy.csv (hunk 3 above)
Comment (rtravis-whoi) [reply deployment/CP04OSPM_Deploy.csv]: Addressed by SNW and this PR looks good to me - approved.

Comment (asmith-whoi) [reply deployment/CP04OSPM_Deploy.csv]: ok

**Pull request merged**